portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/Properties.testcase

```diff
@@ -375,6 +375,139 @@ definition {
 		}
 	}
 
+	@description = "Bug ID: LRAC-8093 | Automation ID: LRAC-11533 | Test Summary: Able to clear property data"
+	@priority = "4"
+	test ClearPropertyData {
+		property analytics.cloud.upstream = "false";
+		property test.name.skip.portal.instance = "Properties#ClearPropertyData";
+
+		var acProperty = "Clear Property";
+
+		task ("Launch AC and go to settings and add a new property") {
+			ACUtils.launchAC();
+
+			ACNavigation.goToSettings();
+
+			ACSettings.goToProperties();
+
+			ACProperties.addNewProperty(propertyName = "${acProperty}");
+
+			ACSettings.takeTokenFromAnalyticsCloud();
+		}
+
+		task ("Connect the DXP to Analytics Cloud") {
+			ACUtils.launchDXP();
+
+			JSONUser.addUser(
+				userEmailAddress = "userea@liferay.com",
+				userFirstName = "userfn",
+				userLastName = "userln",
+				userScreenName = "usersn");
+
+			ACDXPSettings.goToInstanceSettingsAC();
+
+			ACDXPSettings.connectAnalyticsCloud();
+
+			Alert.viewSuccessMessage();
+		}
+
+		task ("Assign the new site to existing property") {
+			ACDXPSettings.navToSyncedSitesMenu();
+
+			ACDXPSettings.assignSiteToExistingProperty(
+				assignedPropertyName = "${acProperty}",
+				siteName = "Site Name");
+		}
+
+		task ("Sync Users to AC") {
+			ACDXPSettings.navToSyncedContactDataMenu();
+
+			ACDXPSettings.syncContactData();
+		}
+
+		task ("Login new user and view page") {
+			User.logoutAndLoginPG(
+				userLoginEmailAddress = "userea@liferay.com",
+				userLoginFullName = "userfn userln");
+		}
+
+		task ("Interact with page") {
+			ACUtils.navigateToSitePageWithUser(
```

Comment:
Hey @joshitagaki, I think this macro would be for you to access the page by passing the email of a user that it would be currently logging in to. I think it should also work to just access the page but to keep the usage pattern you could change it to:

`ACUtils.navigateToSitePage(
		pageName = "AC Page",
		siteName = "Site Name");
`